src/pip/_internal/configuration.py

```diff
@@ -370,7 +370,7 @@ def _get_environ_vars(self):
                 yield key[4:].lower(), val
 
     # XXX: This is patched in the tests.
-    def _iter_config_files(self):
+    def iter_config_files(self):
```

Comment:
I am not sure if this would be helpful, but you may as well create  `iter_config_files()` which calls the `_iter_config_files()`. This way you won't need to modify the private method definition, rather you'd be exposing its functionality via some other public method.

It's just a suggestion. I'll look forward to other opinions on this.

Reply:
That's a good idea. I actually used `iter_config_files()` as per the suggestion from @pradyunsg from the issue https://github.com/pypa/pip/issues/6741#issuecomment-616101245 . 

 IMO we are using this function outside the `pip._internal.Configuration` module just like we are using `load` (we also plan to expose this in `pip debug` output), so it might be better to rename the function then write another wrapper function and increase the maintenance burden. But yes I will too wait for other opinions on this as well

Reply:
There's not much value to add a method that just calls another method, given that we control both the consumers of this API and this API.

Reply:
Agreed. I will just leave the conversion of `_iter_config_files` to `iter_config_files` in this PR as is.